devices/Tlc1543/samples/Tlc1543.Sample.cs

```diff
@@ -0,0 +1,58 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Device.Spi;
+using System.Diagnostics;
+
+namespace Iot.Device.Tlc1543.Samples
+{
+    /// <summary>
+    /// Samples for Tlc1543
+    /// </summary>
+    public class Program
+    {
+        /// <summary>
+        /// Main entry point
+        /// </summary>
+        public static void Main()
+        {
+            SpiDevice spi = SpiDevice.Create(new SpiConnectionSettings(-1) { DataBitLength = Tlc1543.SpiDataBitLength, ChipSelectLine = 5,  });
```

Comment:
Please add a message like this one on SPI setup for devices like ESP32:
https://github.com/nanoframework/nanoFramework.IoT.Device/blob/a3b31ab4efc1dfeed83610a56908c988ce718466/devices/Pn532/samples/Program.cs#L30

In short, it's about adding this:
```csharp
//////////////////////////////////////////////////////////////////////
// when connecting to an ESP32 device, need to configure the SPI GPIOs
// used for the bus
//Configuration.SetPinFunction(21, DeviceFunction.SPI1_MOSI);
//Configuration.SetPinFunction(22, DeviceFunction.SPI1_MISO);
//Configuration.SetPinFunction(23, DeviceFunction.SPI1_CLOCK);
```

Reply:
Also, in this case, -1 won't work, should be a valid one